_posts/2018-04-12-make-a-map.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: D&D Make-A-Map
+date: 2018-04-12
+categories: projects
+description: A web app that randomly generates a D&D Battlemap based on user parameters
+author: Justin Gonzales
+author-image: https://avatars2.githubusercontent.com/u/34696177?s=460&v=4
+author-bio: First Year Computer Security Student at Rochester Institute of Technology
+author-email: jag2724@rit.edu
+author-social:
+  github: https://github.com/justing0927
+---
+# Summary
+D&D Make-A-Map is a web-app built using the Create React App, made to randomly generate d&d maps with various biomes and obstacles with the click of a button (or 3).
```

Comment:
maybe rephrase to `the Create React App library`

Reply:
Maybe the first time you say `D&D` say `Dungeons & Dragons`, for reference for the rest of the post